docs/sources/alerting/set-up/provision-alerting-resources/file-provisioning/index.md

```diff
@@ -0,0 +1,699 @@
+---
+aliases:
+  - /docs/grafana/latest/alerting/provision-alerting-resources/file-provisioning
+  - /docs/grafana/latest/alerting/provision-alerting-resources/file-provisioning
+description: Create and manage resources using file provisioning
+keywords:
+  - grafana
+  - alerting
+  - alerting resources
+  - file provisioning
```

Comment:
Could also do plain `provisioning` as a keyword? Or is that not needed if an existing keyword has multiple words?